src/Components/booksform.js

```diff
@@ -24,15 +24,21 @@ const Booksform = () => {
     dispatch(addBook({
       item_id: bookid,
       ...book,
-      category: 'book',
     }));
   };
 
   return (
-    <form onSubmit={handleSubmit} className="m-3 d-flex justify-content-center align-items-stretch gap-3 flex-direction-row">
-      <input name="title" type="text" placeholder="Booktitle" onChange={handleChange} required />
+    <form onSubmit={handleSubmit} className="m-3 d-flex justify-content-between align-items-stretch gap-3 flex-direction-row">
+      <input style={{ width: '50%' }} name="title" type="text" placeholder="Booktitle" onChange={handleChange} required />
```

Comment:
- [ ] Kindly remove inline styles as this is not best practice.

Reply:
ok i delete and change it to a class